Sophos/manifest.json

```
@@ -38,5 +38,5 @@
     "name": "Sophos",
     "uuid": "0de5216e-19b0-4ad3-9b91-a547cfaf52ca",
     "slug": "sophos",
-    "version": "1.14.0"
+    "version": "1.15.0"
```

For a fix, we prefer to increase the minor part of the version number.
```suggestion
    "version": "1.14.1"
```